packages/react/README.md

```diff
@@ -1,7 +1,128 @@
+<p align="center">
+  <a href="https://clerk.dev?utm_source=github&utm_medium=clerk_react" target="_blank" rel="noopener noreferrer">
+    <img src="https://images.clerk.dev/static/clerk.svg" alt="Clerk logo" height="50">
+  </a>
+  <br />
+</p>
+
 # @clerk/clerk-react
 
-Simple and beautiful user management.
+<div align="center">
+
+[![Chat on Discord](https://img.shields.io/discord/856971667393609759.svg?logo=discord)](https://discord.com/invite/b5rXHjAg7A)
+[![Clerk documentation](https://img.shields.io/badge/documentation-clerk-green.svg)](https://docs.clerk.dev)
+[![Follow on Twitter](https://img.shields.io/twitter/follow/ClerkDev?style=social)](https://twitter.com/intent/follow?screen_name=ClerkDev)
+
+[Changelog](CHANGELOG.md)
+·
+[Report a Bug](https://github.com/clerkinc/javascript/issues/new?assignees=&labels=bug&template=bug_report.md&title=Bug%3A+)
+·
+[Request a Feature](https://github.com/clerkinc/javascript/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=Feature%3A+)
+·
+[Ask a Question](https://github.com/clerkinc/javascript/issues/new?assignees=&labels=question&template=ask_a_question.md&title=Support%3A+)
+
+</div>
+
+---
+
+## Overview
+
+Clerk is the easiest way to add authentication and user management to your React application. Add sign up, sign in, and profile management to your application in minutes.
+
+## Getting Started
+
+### Prerequisites
+
+- React v16+
+- Node.js v14+
+
+### Installation
+
+```sh
+npm install @clerk/clerk-react
+```
+
+### Build
+
+```sh
+npm run build
+```
+
+To build the package in watch mode, run the following:
+
+```sh
+npm run dev
+```
+
+## Usage
+
+Clerk requires your application to be wrapped in the `<ClerkProvider/>` context.
+
+If using Create React App, set `REACT_APP_CLERK_FRONTEND_API` to your Frontend API in your `.env.local` file to make the environment variable accessible on `process.env` and pass it as the `frontendApi` prop.
+
+```jsx
+import { render } from "react-dom";
+
+import {
+  ClerkProvider,
+  SignedIn,
+  SignedOut,
+  SignInButton,
+  UserButton,
+} from "@clerk/clerk-react";
+
+const frontendApi = process.env.REACT_APP_CLERK_FRONTEND_API;
+
+render(
```

Comment:
❓ 
This example seems really unfamiliar to me. The only place I see `render` called directly is in the `index` file of CRA type apps. Can we change it and just add the provider on the `App` component ?
WDYT?

Reply:
It has nothing to do with CRA. `render` is the main ReactDOM method to render a React app into the DOM.